### ISSUE TITLE
how to set video codec H265 for FFmpegFrameRecorder?

### ISSUE THREAD

**Issue by nobitaclash** [bytedeco/javacv#866]
I change the video codec to H265 for faster videostream. The following gives me  org.bytedeco.javacv.FrameRecorder$Exception: avcodec_open2() error -1: Could not open video codec.. 

recorder.setSampleRate(sampleAudioRateInHz);
recorder.setVideoBitrate(videoBitrate);
recorder.setVideoCodec(avcodec.AV_CODEC_ID_H265);
recorder.setPixelFormat(avutil.AV_PIX_FMT_YUV420P);

**Comment by saudet** [comment]
You'll need to provide a way to reproduce your issue. It works just fine for me on Linux:
```
    Stream #0:0(und): Video: hevc (Main) (hev1 / 0x31766568), yuv420p(tv), 402x402 [SAR 3:4 DAR 3:4], 1726 kb/s, 30.06 fps, 30 tbr, 15360 tbn, 30 tbc (default)
```

**Comment by nobitaclash** [comment]
I used FFmpegFrameRecorder in android

**Comment by saudet** [comment]
You're right, it looks like libx265 doesn't work on Android, which I think has already been reported in issues #41 and https://github.com/bytedeco/javacpp-presets/issues/386. Could you check upstream what's up this? I wasn't able to find anything wrong with the build and the same works on Linux.

**Comment by saudet** [comment]
I figured out what the problem was. There was a bug in the patch I created for Android. I've fixed it in the commit above, so please try out new snapshot binaries: http://bytedeco.org/builds/ Thanks!

**Comment by softwarejoint** [comment]
we used ffmpeg in conjuction with realm and webrtc. there has always been an issue with loading of libs with ffmpeg. I saw that you created android 64 bit releases. When will it be possible for you to do a release

**Comment by nobitaclash** [comment]
thanks you for support

**Comment by saudet** [comment]
@softwarejoint Binaries are available on the snapshot server: http://bytedeco.org/builds/

**Comment by saudet** [comment]
Fix included in JavaCV 1.4.1. Thanks for reporting and enjoy!

**Comment by max-tian** [comment]
It's not been fixed in version 1.5.2 JavaCV. The settings as below.
`
recorder.setVideoCodec( avcodec.AV_CODEC_ID_H265 );
recorder.setPixelFormat( avutil.AV_PIX_FMT_YUV420P );
`

**Comment by saudet** [comment]
@max-tian I'm pretty sure that works, but please first update to JavaCV 1.5.7 just to be sure.
